build.ps1

```diff
@@ -51,10 +51,20 @@ $ToolsFolder = Join-Path $ProjectRoot tools
 $BinFolder = Join-Path $ProjectRoot "bin"
 $BinConfigFolder = Join-Path $BinFolder $Configuration
 $BinTestsFolder = Join-Path $BinConfigFolder "tests"
-$PackageRestoreRoot = Join-Path $env:userprofile '.nuget/packages/'
+
+if ($IsWindows) {
```

Comment:
This variable is not defined in Powershell on Windows. 
Perhaps you can just check whether `$env:userprofile` is defined and use it if it is, otherwise fallback to `$env:home`.